src/enums/clip_action.ts

```diff
@@ -0,0 +1,6 @@
+enum ClipAction {
+    NEW_CLIP = "new_clip",
+    REPLAY = "replay_clip",
```

Comment:
Do we still have a replay action?

Reply:
This enum value is passed when the playSong is called for replays